packages/sanity/test/cli/schema/listSchemasAction.test.ts

```diff
@@ -31,6 +31,7 @@ const workspace3 = testMultiWorkspaceManifest.workspaces[2]
 const validStoredSchema1: StoredWorkspaceSchema = {
   _id: getWorkspaceSchemaId({workspaceName: workspace1.name}).safeId,
   _type: 'sanity.workspace.schema',
+  version: 2,
```

Comment:
This is a bit of a paranoia addition. I suspect we might need to check for this once the new Canvas link mechanism is done. This branch needs to go out at the same time as that feature.